data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,86 @@ mission "Station Exploration Abandon Ship"
+			branch remnant
+				has "outfit: Quantum Key Stone"
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label remnant
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "I have noticed that the Keystone your ship bears is from beyond the wormhole. If your people have access to Keystones as well, then I hope they too become wise to its power. Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+			label choice
+			choice
+				`	"I can take you there."`
+					goto agree
+				`	"Why would you want to go to <destination>? Isn't that an Unfettered planet?"`
+					goto ask
+					to display
+						or
+							has "visited system: Wah Ki"
+							has "Wanderers: Translation Machine: offered"
+							has "First Contact: Unfettered: offered"
+				`	"I'm not going there. You'll have to ask someone else."`
+					decline
+			label ask
+			`	The Hai pauses for a moment, as if unsure of what to say. "My teachings and prophecies are more... appreciated among our northern brethren."`
+			choice
+				`	"Teachings?"`
+				`	"Alright, I'll take you there."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			label teachings
+			`	An unreadable expression appears on Hareek's face. "Ah, apologies for my rudeness - you probably do not know who I am. How do I explain this... You have your preachers and wise men among the humans, yes? I am something similar. Now, will you take me to <destination>?"`
+			choice
+				`	"So you're going to preach to the Unfettered?"`
+				`	"I can take you."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			`	"That is an oversimplification, but it is not something I can explain adequately to a human." He turns away, looking into the fog. "There are portentous events occurring in the galaxy, and I must warn others of the turmoil I have foreseen. The Unfettered crave turmoil, so naturally they are more inclined to believe me." He gives a wry smile.`
```

Comment:
Shouldnt this only trigger before the invasion? Like, I imagine he'd find his way there without you otherwise
Or mb just add that some already happened if its after that, and they need his council?

Reply:
I deliberately kept it vague as to what great event he was talking about, so that it could be applicable to Hai Reveal, the Wanderer campaign, the upcoming Unfettered campaign, etc.

Reply:
That's a good point, that this could be read as applying to whatever -may- happen next, even after everything else directly affecting Unfettered in the game at any point in time is completed.